stratum/hal/bin/barefoot/docker/stratum_profile.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+# SDE_VERSION : 9.2.0
+
+stratum_profile:
```

Comment:
Do we still need this? I think we have a more universal profile that builds PI and BfRt for Tofino and Tofino2

Reply:
/stratum/hal/bin/barefoot/stratum_profile.yaml